src/shared/components/VideoTileBase/VideoTileBase.styles.tsx

```diff
@@ -201,7 +201,7 @@ export const ProgressBar = styled.div`
   background-color: ${colors.blue['500']};
 `
 
-export const CoverVideoPublishingStateOverlay = styled.div`
+export const VideoHeroPublishingStateOverlay = styled.div`
```

Comment:
I don't think this is a desired change, this doesn't have anything to do with the `VideoHero` component